scripts/upload_whatsapp_templates/upload_whatsapp_templates.py

```diff
@@ -0,0 +1,131 @@
+import argparse
+import json
+from csv import DictReader
+from urllib.parse import urljoin
+
+import attr
+import requests
+
+
+def length_limit(limit):
+    def validate_length(instance, attribute, value):
+        if len(value) > 1024:
```

Comment:
This should probably use the limit var

Reply:
I also see instance is not being used but it's probably the signature required by the validator.

Reply:
Whoops, good catch. Yeah it's required by the validator.